lib/jackalope.ex

```diff
@@ -252,10 +193,16 @@ defmodule Jackalope do
     # Default backoff options is 1 sec to 30 secs, doubling each time.
     backoff_opts = Keyword.get(opts, :backoff) || [min_interval: 1_000, max_interval: 30_000]
     Logger.info("[Jackalope] Connecting with backoff options #{inspect(backoff_opts)}")
+    initial_topics = Keyword.get(opts, :initial_topics)
```

Comment:
Since this is static and now the only place to specify subscriptions, should we just call this `topics` ? (Maybe support both for now?)

Also, should we validate this more now as a required option? I can't think of a use case where we'd start jackalope and _not_ subscribe to anything and it seems like we can help protect against that with some failure/warning?